ztrouy/frontend-capstone-api
# Feature - Managing Game Ownership

### Purpose
To allow Users to specify whether or not they own a Game, and change that state at any time

### Files Changed
- Wrote a view that manages the ownership status of a Game for a User that accepts POST and DELETE requests in `games.py`

### To Test
Fetch, setup by running `pipenv shell` and `./setup_database.sh` in the project root directory, run via the debugger, then confirm the following
- POST request to `/games/{id}/own` creates UserGame entry for the appropriate User and Game, if the Game exists and an equivalent UserGame entry does not already exist
- DELETE request to `/games/{id}/own` removes the appropriate UserGame entry if it exists
- After creating or removing the UserGame entry, the "isOwned" property of the related Game accurately reflects the change when fetched

closes #17
